### ISSUE TITLE
RAK11720 compile errors with AS923 disabled

### ISSUE THREAD

**Issue by JavierNR** [RAKWireless/RAKwireless-Arduino-BSP-Index#9]
Hello,

First of all, thank you for implement timereq commands and callbacks. I've been testing 4.1.1_279 from staging and I have a little hiccup when disabling all LoRaWAN regions except EU868. Board RAK11720
The compiler throws the following error:

````
c:/users/javie/appdata/local/arduino15/packages/rak_rui/tools/arm-none-eabi-gcc/9-2019q4/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld.exe: C:\Projects\GEM RUI3\GEM-V2-Firmware\build/core\core.a(service_lora.c.o): in function `service_lora_init':
C:\Users\javie\AppData\Local\Arduino15\packages\rak_rui\hardware\apollo3\4.1.1\cores\apollo3\component\service\lora/service_lora.c:754: undefined reference to `RegionAS923SubBandSet'
c:/users/javie/appdata/local/arduino15/packages/rak_rui/tools/arm-none-eabi-gcc/9-2019q4/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld.exe: C:\Users\javie\AppData\Local\Arduino15\packages\rak_rui\hardware\apollo3\4.1.1\cores\apollo3\component\service\lora/service_lora.c:769: undefined reference to `RegionAS923SubBandSet'
c:/users/javie/appdata/local/arduino15/packages/rak_rui/tools/arm-none-eabi-gcc/9-2019q4/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld.exe: C:\Projects\GEM RUI3\GEM-V2-Firmware\build/core\core.a(service_lora.c.o): in function `service_lora_set_band':
C:\Users\javie\AppData\Local\Arduino15\packages\rak_rui\hardware\apollo3\4.1.1\cores\apollo3\component\service\lora/service_lora.c:1446: undefined reference to `RegionAS923SubBandSet'
c:/users/javie/appdata/local/arduino15/packages/rak_rui/tools/arm-none-eabi-gcc/9-2019q4/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld.exe: C:\Users\javie\AppData\Local\Arduino15\packages\rak_rui\hardware\apollo3\4.1.1\cores\apollo3\component\service\lora/service_lora.c:1458: undefined reference to `RegionAS923SubBandSet'
````

I'm bypassing it by temporally enabling Region AS923.

Regards

**Comment by beegee-tokyo** [comment]
Thanks for reporting.
Send the issue to R&D team. Should be fixed in official release of RUI3 V4.1.1

**Comment by JavierNR** [comment]
Hello,

I can confirm the issue as solved with last version.
Thanks again
